src/code/z_scene_table.c

```diff
@@ -995,8 +1013,8 @@ void func_80099760(GlobalContext* globalCtx) {
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_scene_table.c", 4859);
 }
 
-u32 D_8012A300[] = { 0x02012378, 0x02013378 };
-u32 D_8012A308[] = { 0x02011F78, 0x02014778, 0x02014378, 0x02013F78, 0x02014B78, 0x02013B78, 0x02012F78, 0x02012B78 };
+void* gDCEntranceTextures[] = { gDCDayEntranceTex, gDCNightEntranceTex };
+void* sDCLavaFloorTextures[] = { gDCLavaFloor1Tex, gDCLavaFloor2Tex, gDCLavaFloor3Tex, gDCLavaFloor4Tex, gDCLavaFloor5Tex, gDCLavaFloor6Tex, gDCLavaFloor7Tex, gDCLavaFloor8Tex };
```

Comment:
This looks like it needs to be re-formatted, and I suggest adding trailing commas to all these texture pointer arrays to improve the formatting and make it consistent.